django/utils/dateformat.py

```diff
@@ -325,8 +325,8 @@ def W(self):
         return self.data.isocalendar()[1]
 
     def y(self):
-        "Year, 2 digits; e.g. '99'"
-        return str(self.data.year)[2:]
+        "Year, 2 digits; e.g. '99', with leading zero"
+        return ("000" + str(self.data.year))[-2:]
```

Comment:
What do you think about?
```suggestion
        return '%02d' % (self.data.year % 100)
```

Reply:
Right, that is much nicer. Actually, what about f-strings?
```f'{self.data.year % 100:02}'```

Reply:
We allowed them but without inline calculations (see [Python style](https://docs.djangoproject.com/en/3.1/internals/contributing/writing-code/coding-style/#python-style)) so I would use `%`-formatting in this case.